backend/src/utils/fileUtils.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { KubeFastifyInstance } from '../types';
+
+export const writeAdminLog = (fastify: KubeFastifyInstance, data: object): void => {
```

Comment:
I recommend you type `data` -- but it's a logging feature -- so I guess if you don't care if it's uniform, it's fine.

Reply:
```
/home/runner/work/odh-dashboard/odh-dashboard/backend/src/utils/fileUtils.ts
  5:67  error  Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys  @typescript-eslint/ban-types

✖ 1 problem (1 error, 0 warnings)
```

Turns out -- you actually have a linter error. Our test system does not do well with backend issues -- it spawns the frontend version through success or fails. Awkward.